x/compute/internal/types/errors.go

```diff
@@ -2,39 +2,59 @@ package types
 
 import (
 	sdkErrors "github.com/enigmampc/cosmos-sdk/types/errors"
+	"strings"
 )
 
 // Codes for wasm contract errors
 var (
 	DefaultCodespace = ModuleName
 
 	// ErrCreateFailed error for wasm code that has already been uploaded or failed
-	ErrCreateFailed = sdkErrors.Register(DefaultCodespace, 1, "create wasm contract failed")
-
-	// ErrAccountExists error for a contract account that already exists
-	ErrAccountExists = sdkErrors.Register(DefaultCodespace, 2, "contract account already exists")
+	ErrCreateFailed = sdkErrors.Register(DefaultCodespace, 1, "create contract failed")
 
 	// ErrInstantiateFailed error for rust instantiate contract failure
-	ErrInstantiateFailed = sdkErrors.Register(DefaultCodespace, 3, "instantiate wasm contract failed")
+	ErrInstantiateFailed = sdkErrors.Register(DefaultCodespace, 2, "instantiate contract failed")
 
```

Comment:
Why did you reshuffle these lines?

Reply:
So IsEncryptedError can check the error code by checking 0 < x < 5 instead of matching random values

Reply:
ahhhh, cool cool cool. I'd add a comment about it in this section of the code just to avoid subtle bugs if anyone adds to this list.